ui/cypress/e2e/collectors.test.ts

```diff
@@ -121,11 +123,12 @@ describe('Collectors', () => {
       const secondTelegraf = 'test2'
       const thirdTelegraf = 'unicorn'
       const description = 'Config Description'
+      const secondBucket = 'bucket2'
 
       cy.get('@org').then(({id}: Organization) => {
-        cy.createTelegraf(firstTelegraf, description, id)
-        cy.createTelegraf(secondTelegraf, description, id)
-        cy.createTelegraf(thirdTelegraf, description, id)
+        cy.createTelegraf(firstTelegraf, description, id, bucket)
+        cy.createTelegraf(secondTelegraf, description, id, bucket)
+        cy.createTelegraf(thirdTelegraf, description, id, secondBucket)
```

Comment:
I would create all of these before this test runs in a `beforeEach`.  You can then nest the sort by name and sort by bucket tests and not have to repeat this creation logic.

Reply:
```
describe('sorting', () => {
  beforeEach(() => {
    // creation logic here
  }
  // sort by buckets test here
  // sort by name test here
}
```